static/app/views/metrics/chart/chart.tsx

```diff
@@ -338,13 +339,14 @@ export const MetricChart = memo(
         focusArea,
         releases,
         firstUnit,
+        additionalSeries,
       ]);
 
       if (!enableZoom) {
         return (
           <ChartWrapper>
             {focusArea?.overlay}
-            <CombinedChart {...chartProps} additionalSeries={additionalSeries} />
+            <CombinedChart {...chartProps} />
```

Comment:
shouldn't that change alone fix the conflict?
So, there is no need to pass it as `series`?